patterns/gitops/multi-cluster-hub-spoke-argocd/spokes/main.tf

```
@@ -252,18 +254,28 @@ module "eks" {
 
   vpc_id     = module.vpc.vpc_id
   subnet_ids = module.vpc.private_subnets
-
-  manage_aws_auth_configmap = true
-  aws_auth_roles = [
-    # Granting access to ArgoCD from hub cluster
-    {
-      rolearn  = aws_iam_role.spoke.arn
-      username = "gitops-role"
-      groups = [
-        "system:masters"
-      ]
-    },
-  ]
+  
+  authentication_mode = "API"
```

Please update this to use the variable
```suggestion
  authentication_mode = local.authentication_mode
```